contrib/kind.sh

```diff
@@ -216,40 +180,71 @@ ovn_apiServerAddress=${API_IP} \
 kind create cluster --name ${KIND_CLUSTER_NAME} --kubeconfig ${HOME}/admin.conf --image kindest/node:${K8S_VERSION} --config=${KIND_CONFIG_LCL}
 export KUBECONFIG=${HOME}/admin.conf
 cat ${KUBECONFIG}
-mkdir -p /tmp/kind
-sudo chmod 777 /tmp/kind
-count=0
-until kubectl get secrets -o jsonpath='{.items[].data.ca\.crt}'
-do
-  if [ $count -gt 10 ]; then
-    echo "Failed to get k8s crt/token"
-    exit 1
-  fi
-  count=$((count+1))
-  echo "secrets not available on attempt $count"
-  sleep 5
-done
-kubectl get secrets -o jsonpath='{.items[].data.ca\.crt}' > /tmp/kind/ca.crt
```

Comment:
I've removed these certificates and extramounts because it seems they are not needed when running in a pod, however,  please check  my logic is correct.

Reply:
I haven't seen any issues here. Assuming it is OK.

Reply:
this was needed in the past for one of the pods to be able to use the API server IIRC. I should have put a comment what needed this, but if it is no longer needed then that's fine to remove.